src/components/AIrelated/ChatButton.tsx

```
@@ -97,63 +134,96 @@ const ChatButton = () => {
-                  className="max-w-[80%] p-2 rounded-lg text-sm"
+                  className={cn(
+                    "max-w-[85%] rounded-lg",
+                    isExpanded ? "p-4 text-base" : "p-2 text-sm"
+                  )}
                   style={{
                     backgroundColor: msg.sender === 'user' ? theme.colors.primary : theme.colors.tertiary,
                     color: msg.sender === 'user' ? theme.colors.background : theme.colors.text
                   }}
                 >
-                  <p>{msg.text}</p>
+                  <p dangerouslySetInnerHTML={{ 
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)